--- /home/weiran/Desktop/linux/drivers/mtd/spi-nor/gigadevice.c.orig	2021-05-19 19:02:23.354521257 -0700
+++ /home/weiran/Desktop/linux/drivers/mtd/spi-nor/gigadevice.c	2021-05-19 19:05:04.094300590 -0700
@@ -49,7 +49,7 @@ static const struct flash_info gigadevic
 			   SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ |
 			   SPI_NOR_4B_OPCODES | SPI_NOR_HAS_LOCK |
 			   SPI_NOR_HAS_TB | SPI_NOR_TB_SR_BIT6)
-		 .fixups = &gd25q256_fixups},
+		 },
 };
 
 const struct spi_nor_manufacturer spi_nor_gigadevice = {
